### DISCUSSION TITLE
How to make submenus

### DISCUSSION THREAD

**Discussion by mohitKS777** [chakra-ui/chakra-ui#6347] [Q&A]
I am working with chakra ui menus, and want to make submenus like discussed [here](https://twitter.com/thesegunadebayo/status/1257280343092191233?lang=en). But there is nothing mentioned about submenus in the docs, and the example used in this twitter thread doesn't work as expected. Does chakra supports submenus or do I have make them somehow myself.

**Comment by timkindberg** [comment]
https://github.com/chakra-ui/chakra-ui/discussions/3671#discussioncomment-564990
